### DISCUSSION TITLE
Not Able to Side-scroll with Overflow

### DISCUSSION THREAD

**Discussion by ghost** [requarks/wiki#3677] [Error / Bug Report]
**Describe the bug**
One of my pages has a large HTML table that has 7 columns. When I go to view the page, it only shows 5.5 of the columns (it shows half of the 6th column and does not show the 7th column). When I try to side scroll, it only lets me scroll down the page.

I tried adding overflow: scroll to my table CSS, but that only worked as expected in the editor and not when the page was published.

**To Reproduce**
1. Create a table with 8 columns
2. Make the column widths large enough so that the far right columns go off the page
3. Publish
4. Check that you can't side-scroll

**Expected behavior**
You should be able to side scroll to the end of the table to see the information. It seems like there is overflow:hidden set somewhere.

**Host Info (please complete the following information):**
 - OS: Ubuntu Linux (xenial) 16.04 x64
 - Wiki.js version: 2.2.51
 - Database engine: postgres 9.5.21

**Comment by Smankusors** [comment]
is this still happen with Wiki.js 2.3.81?

**Comment by ghost** [comment]
> is this still happen with Wiki.js 2.3.81?

Just upgraded to 2.3.81. Still an issue.

**Comment by pierreguilmin** [comment]
I am having the same issue on Chrome and Firefox. Setting the following custom css for the `table` tag fixes this:
```css
table {
  display: block;
  overflow-x: auto;
}
```

Is this something we could fix easily @NGPixel?

**Comment by stukev** [comment]
This is still an issue. Fixed it on my instance with a CSS fix.
